l10n_it_intrastat_statement/tests/test_intrastat_statement.py

```diff
@@ -163,6 +163,30 @@ def test_statement_sale_quarter(self):
         # Last line is section line, for quarter report it should be 64 chars
         self.assertEqual(len(file_content.splitlines()[-1]), 64)
 
+    def test_statement_sale_quarter_adjustment_period(self):
+        """Check that sale adjustments can be in a different period type"""
+        invoice = self._get_intrastat_computed_invoice()
+        month = fields.Date.from_string(invoice.date_invoice).month
+        quarter = 1 + (month - 1) // 3
```

Comment:
Operator `//` in python2 exists?

Reply:
Yes: https://docs.python.org/2.7/reference/lexical_analysis.html#operators

Reply:
Thanks